src/modules/ui/components/layout/index.js

```diff
@@ -3,6 +3,9 @@ import React from 'react';
 import VSplit from './vsplit';
 import HSplit from './hsplit';
 import SplitPane from '@kadira/react-split-pane';
+import FuzzySearch from 'react-fuzzy';
```

Comment:
I like to move search component into a separate component and container by moving all these code out from the layout. Like I did for [shortcuts_help](https://github.com/kadirahq/storybook-ui/blob/master/src/modules/ui/containers/shortcuts_help.js).

And then simply use them inside the routing code. See: https://github.com/jtangelder/sass-loader